src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -404,19 +404,24 @@
     <Field Name="Right" Type="BoundExpression"/>
 
     <Field Name="DeconstructSteps" Type="ImmutableArray&lt;BoundDeconstructionDeconstructStep&gt;" Null="NotApplicable" SkipInVisitor="true" />
```

Comment:
@gafter  tuple conversions already store and can be asked for the underlying conversions. 
+1 for reviewing the bound nodes, but not right now.